python-sdk/src/astro/utils/table.py

```diff
@@ -101,7 +101,7 @@ def find_first_table(
 
     if op_args:
         first_table = _find_first_table_from_op_args(op_args=op_args, context=context)
-    if not first_table and op_kwargs and python_callable:
+    if not first_table and op_kwargs and python_callable is not None:
```

Comment:
@pankajastro Why did we add this?

Reply:
otherwise the static check is failing since `python_callable` which is not boolean

Reply:
Interesting 🤔 🤷

Reply:
also, I feel it does make sense if we pass empty callable 

```
val = [ ]
if val:   => won't go in this block
    print("fail") 
if val is not None:
    print("pass")
```

Reply:
```
if val:   => won't go in this block
    print("fail") 
```
this is expected